cmd/operator/main.go

```diff
@@ -174,6 +178,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Get and parse the worker ignition spec. Done once on operator start.
```

Comment:
We should call out somewhere that we are changing the behavior when compared to WMCB when we were doing it once per node.

Reply:
Changed this as it was unnecessary.